static/js/admin/src/action/ClearAccessSurrogateAction.js

```diff
@@ -0,0 +1,66 @@
+define('ClearAccessSurrogateAction', [ 'jquery', 'AjaxCallbackAction'], function($, AjaxCallbackAction) {
+
+    function ClearAccessSurrogateAction(context) {
+        this._create(context);
+    }
+
+    ClearAccessSurrogateAction.prototype.constructor = ClearAccessSurrogateAction;
+    ClearAccessSurrogateAction.prototype = Object.create( AjaxCallbackAction.prototype );
+
+    ClearAccessSurrogateAction.prototype._create = function(context) {
+        this.context = context;
+
+        var options = {
+            workMethod: "DELETE",
+            workPath: "/services/api/edit/accessSurrogate/{idPath}",
+            workLabel: "Clearing assigned access surrogate...",
+            followupLabel: "Clearing assigned access surrogate...",
+            followupPath: "/services/api/status/item/{idPath}/solrRecord/version"
+        }
+
+        if ('confirm' in this.context && !this.context.confirm) {
+            options.confirm = false;
+        } else {
+            options.confirm = {
+                promptText : "Clear the assigned access surrogate?",
+                confirmAnchor : this.context.confirmAnchor
+            };
+        }
+
+        AjaxCallbackAction.prototype._create.call(this, options);
+    };
+
+    ClearAccessSurrogateAction.prototype.completeState = function() {
+        this.context.actionHandler.addEvent({
+            action : 'RefreshResult',
+            target : this.context.target
+        });
+        this.alertHandler.alertHandler("success", "Cleared the assigned access surrogate.");
+        this.context.target.enable();
+    };
+
+    ClearAccessSurrogateAction.prototype.workDone = function(data) {
+        this.completeTimestamp = data.timestamp;
+        this.oldAccessSurrogateId = data.oldAccessSurrogateId;
```

Comment:
same question here, what is `oldAccessSurrogateId`? I don't think the access surrogates have ids since they are just datastreams of the file object, so I don't think there would be an id that change. Some of the details of the datastream might change, but I'd guess the timestamp/version of the file object to be the main way to tell if the work is done